### ISSUE TITLE
How to reduce interval of checking for new image from the registry

### ISSUE THREAD

**Issue by st3fus** [argoproj-labs/argocd-image-updater#602]
Hey there, so I got everything working for several weeks now, and accidentally I noticed this on my docker hub for both repos I'm using:

<img width="111" alt="image" src="https://github.com/argoproj-labs/argocd-image-updater/assets/112018215/be107283-52ea-416c-a9e7-f1ab6a32b00a">

How do I reduce the interval it checks/pulls latest image? I did set limit: 5 in repo config but I think it's still checking it too many times. I do have a paid dockerhub license so I can pull 5000times per day without hitting a limit, but still.

**Comment by jwhy89** [comment]
This can be set in the extra args in the helm chart values:
![CleanShot 2023-08-08 at 13 46 06](https://github.com/argoproj-labs/argocd-image-updater/assets/45811933/9c9c691d-df3f-4836-b4fe-1846c0580562)

What I think you've configured is the rate at which it tries to fetch the actual data from source registry.

**Comment by jwhy89** [comment]
@st3fus see above

**Comment by st3fus** [comment]
> This can be set in the extra args in the helm chart values: ![CleanShot 2023-08-08 at 13 46 06](https://user-images.githubusercontent.com/45811933/259199039-9c9c691d-df3f-4836-b4fe-1846c0580562.png)
> 
> What I think you've configured is the rate at which it tries to fetch the actual data from source registry.

Which helm chart exactly? The ones from the backend and front or the main chart?

**Comment by jwhy89** [comment]
> > This can be set in the extra args in the helm chart values: ![CleanShot 2023-08-08 at 13 46 06](https://user-images.githubusercontent.com/45811933/259199039-9c9c691d-df3f-4836-b4fe-1846c0580562.png)
> > What I think you've configured is the rate at which it tries to fetch the actual data from source registry.
> 
> Which helm chart exactly? The ones from the backend and front or the main chart?

The main chart in the values.yaml file.

**Comment by st3fus** [comment]
Seems like I installed it with default helm values, can I download the default values.yaml and have only the extraArgs in it? I'm guessing when I do kubectl -f apply on that it will only modify the extraArgs since I only have that inside values.yaml?

**Comment by jwhy89** [comment]
> Seems like I installed it with default helm values, can I download the default values.yaml and have only the extraArgs in it? I'm guessing when I do kubectl -f apply on that it will only modify the extraArgs since I only have that inside values.yaml?

Yep, that or pass in flags like https://github.com/helm/charts/issues/2112#issuecomment-370650234

**Comment by st3fus** [comment]
@jwhy89 I just got back to this issue, was busy with other projects... I've set it up all on my previous PC, now that I switched to a new one I can't recall how I installed it... I think I used helm, but helm list doesn't output anything on this PC. I got kubectl and argoCD, I just want to modify that extraArgs so I can increase the interval of checks.

I'm not exactly sure how should I apply that setting without messing up my current deployment.
